libcxx/docs/Status/SpaceshipPapers.csv

```diff
@@ -1,5 +1,5 @@
 "Number","Name","Status","First released version"
```

Comment:
Could we actually get rid of this file? I don't think it serves much of a purpose anymore. We should make sure that all the information in that file is tracked in our other `Cxx20Papers.rst` & friends and then get rid of it.

Reply:
This page has been shipped in several libc++ releases. I strongly like to have it shipped in one release where P1614 is finished. I already have a TODO on my private list to remove it after branching LLVM 20.